app/controllers/providers/update_email_addresses_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Providers
+  class UpdateEmailAddressesController < ProviderBaseController
+    def show
+      authorize legal_aid_application
+      @form = Applicants::BasicDetailsForm.new(model: applicant)
```

Comment:
Creating a new form would make sense I think. Because on this page we are only editing the email.

Reply:
There actually is an email form already so I have used that.

Reply:
`Applicants::BasicDetailsForm` is not an email form but more a form for applicant's details which happens to contains `email`. 
The risk in using this form just for the email is that it could potentially trigger some `presence` validation errors on the other attributes.

So it would make sense to create another form specific to this page (i.e. `Applicants::EmailForm`).

On the other hand, because this new email page is always after the applicant's details page, the other attributes (date of birth, NIN, first_name, last_name) should always be populated and valid so no other errors than emails errors should be raised.
So I guess it's fine to reuse this same form even though it's a bit strange to re-validate attributes that are not on the page.

Reply:
Oops. I think I misunderstand your previous comment. I see now that you are re-using an already existing `Applicants::EmailForm`. I didn't realise there was already one